cloudbuild.yaml

```diff
@@ -25,81 +28,299 @@ steps:
   - container
   - clusters
   - get-credentials
-  - "limani-integ"
+  - cluster-1
```

Comment:
Cluster and zone should be env variables. These can be set up as part of the cloud build trigger.